index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
+            <a>AudioWorkletProcessor</a>
+          </h2>
+          <p>
+            When the constructor of <a>AudioWorkletNode</a> is invoked in the
+            main global scope, the corresponding <a>AudioWorkletProcessor</a>
+            instance is automatically created in
+            <a>AudioWorkletGlobalScope</a>. After the construction, they
+            maintain the internal reference to each other until the
+            <a>AudioWorkletNode</a> instance is destroyed.
+          </p>
+          <p>
+            When <code><a>AudioWorkletNode</a>(<em>context</em>, <em>name</em>,
+            <em>options</em>)</code> constructor is called, the user agent must
+            run the following steps in a synchronous fashion:
+          </p>
+          <ol>
+            <li>If the <code><em>context</em></code> is not an instance of <a>
+              BaseAudioContext</a>, throw a <code>TypeError</code> and abort
+              all these steps.
+            </li>
+            <li>If the <code><em>name</em></code> is not a valid
+            <code>DOMString</code> or does not exists as a key in the
+            <a>AudioWorkletGlobalScope</a>’s <a>node name to processor
+            definition map</a>, throw a <code>NotSupportedError</code>
+            exception and abort all these steps.
+            </li>
+            <li>Let <code><em>processorConstructor</em></code> be the result of
+            looking up name on the <a>AudioWorkletGlobalScope</a>’s <a>node
+            name to processor definition map</a>.
+            </li>
+            <li>Let <code><em>baseNodeInstance</em></code> be a new instance of
+            <a>AudioWorkletNode</a>.
+            </li>
+            <li>Let <code><em>options</em></code> be the optional
+            <code>options</code> argument in the constructor.
+            </li>
+            <li>Let <code><em>processorInstance</em></code> be a new instance
+            of <a>AudioWorkletProcessor</a> created by processorConstructor.
+            </li>
+            <li>Let <code><em>parameterDescriptors</em></code> be the result of
+            <code><a href=
+            "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=processorConstructor,
+            P="parameterDescriptors")</code>. If
+            <code><em>parameterDescriptors</em></code> in
+            <code>undefined</code>, skip to the step 9.
+            </li>
+            <li>For each <code><em>descriptor</em></code> in
+            <code><em>parameterDescriptors</em></code> perform the following
+            substeps:
+              <ol>
+                <li>Let <code><em>audioParam</em></code> be a new
+                <a>AudioParam</a> instance with:
+                  <ul>
+                    <li>
+                      <code><em>paramName</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
```

Comment:
For `item` here and in the following few lines, I think you mean `descriptor` which is the iteration variable above.